server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/redirect/NotifyMailetsMessage.java

```diff
@@ -80,8 +82,11 @@ public String generateMessage(String parameterMessage, Mail originalMail) throws
         appendAddresses(builder, "To", message.getHeader(RFC2822Headers.TO));
         appendAddresses(builder, "CC", message.getHeader(RFC2822Headers.CC));
 
-        builder.append("  Size: " + humanSize(message.getSize()))
-            .append(LINE_BREAK);
+        getMessageSizeEstimation(originalMail).ifPresent(size -> {
+            builder.append("  Size: " + humanSize(size));
+            builder.append(LINE_BREAK);
```

Comment:
you can get ride of the {} with size -> builder.append("  Size: " + humanSize(size))
.append(LINE_BREAK)

Reply:
done